html.cfg

```diff
@@ -8,11 +8,26 @@
 }}
 
 \Css{body {
-    max-width: 50rem;
+    max-width: 55rem;
     box-sizing: border-box;
     padding: 1rem;
     margin: 0 auto;
     overflow-x: hidden;
+    background-color: #F4ECD8;
```

Comment:
Add '\\' before character '#', otherwise `make4ht` will see it as escape character and raise error.